composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "atlas/pdo",
     "type": "library",
+    "version": "2.0.0",
```

Comment:
Composer recommends against adding the version; it will get picked up automatically from the branch name.

Reply:
I saw the warning when composer run the update... :) 

Removed it. Thanks